### PR TITLE
Decode a Morse code message

### PR DESCRIPTION
**Pull request by gdvl777**
Description
This pull request introduces a series of methods that fulfill the following requirements:

General Requirements

- [x]  No linter errors.
- [x]  Gitflow was used.
- [x]  Work was documented in a professional manner.

Ruby Requirements
 Best practices for Ruby were followed.
Project Requirements

- [x]  Decode a Morse Character: A method was created to decode a Morse character, taking a string and returning the corresponding character in uppercase.
- [x]  Decode a Morse Word: A method was created to decode an entire word in Morse, taking a string and returning the string representation.
- [x]  Decode an Entire Morse Message: A method was created to decode an entire Morse message, taking a string and returning the string representation. Each word is separated by 3 spaces.
- [x]  Reuse of Methods: Ensured that the code is DRY by reusing methods wherever possible.
- [x]  Decoding a Real Message: The code was used to decode the message from an old bottle: .- -... --- -..- ..-. ..- .-.. .-.. --- ..-. .-. ..- -... .. . ....

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,291 +1,156 @@
...
+---
 
 <!-- LICENSE -->
 
 ## 📝 License <a name="license"></a>
 
-This project is [MIT](./LICENSE) licensed.
-
-_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
+This project is [MIT](./MIT) licensed.
```
Comment (elisha2kyakpo1) [inline README.md]: - [x] Kindly fix the license link which is giving a 404 error. The readme file shouldn't have broken links for it to be professional.

**Review by elisha2kyakpo1 — CHANGES_REQUESTED** [review]
Hi 👋 @gdvl777, @ITurres, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️ The PR title and description are good and professional.
✔️ No linter errors.
✔️ Descriptive commit messages.
👏 The app works as expected.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (gdvl777) [reply README.md]: You had reviewed an incorrect readme. I mean, you viewed the readme file in dev branch. We worked on feature/morse_code branch

**Review by aamirkhan2478 — APPROVED** [review]
Hi @gdvl777 & @ITurres, 

GIF

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✅ You used Gitflow correctly :+1: 
✅ No linter error :+1: 
✅ Good commit messages :+1: 
✅ Good descriptive PR :+1: 
✅ Well done :clap: you did all changes given by the previous reviewer :+1: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ITurres** [comment]
Hi, @aamirkhan2478 and @elisha2kyakpo1! Thank you or having reviewed this PR! 💯 , I wish you a great week and happy coding!

**Pull request merged**